### PR TITLE
Feature/newsletter add image

### PR DESCRIPTION
**Pull request by CorvinAlex**
PLZ test pass

### CODE REVIEW

File: app/Http/Controllers/Api/NewsletterController.php [hunk 1]
```diff
@@ -61,6 +64,19 @@ private function bindNewsletterAndCustomer($customerIds, $newsletter, $user)
         }
     }
 
+    private function bindNewsletterAndArtwork($artworkIds, $newsletter, $user)
```
Comment (DeseineClement) [inline app/Http/Controllers/Api/NewsletterController.php:67]: you sure you want to pass an array in a route ?

File: app/Http/Controllers/Api/NewsletterController.php [hunk 2]
```diff
@@ -61,6 +64,19 @@ private function bindNewsletterAndCustomer($customerIds, $newsletter, $user)
...
+    {
+        foreach ($artworkIds as $artworkId) {
+            $artwork = Artwork::find($artworkId);
+            if (is_null($artwork) || ($artwork->gallery->id != $user->gallery->id)) {
```
Comment (DeseineClement) [inline app/Http/Controllers/Api/NewsletterController.php]: use Isset instead

File: app/Http/Controllers/Api/NewsletterController.php [hunk 3]
```diff
@@ -188,4 +212,28 @@ public function send(Newsletter $newsletter)
...
+     * @param Newsletter $newsletter
+     *
+     * @return \Illuminate\Http\JsonResponse
+     */
+    public function preview(Newsletter $newsletter)
+    {
+        if ($newsletter->gallery->id != request()->user()->gallery->id) {
+            throw new UnauthorizedException('The current gallery does not own this newsletter.');
+        }
+        $customer = $newsletter->customers()->get()[0];
```
Comment (DeseineClement) [inline app/Http/Controllers/Api/NewsletterController.php]: why not using first() instead ?

File: app/Http/Controllers/Api/UserController.php [hunk 4]
```diff
@@ -155,6 +182,10 @@ public function storeMember(MemberStoreRequest $request)
      */
     public function show(User $user)
     {
+        /*Mail::send('email.subscriptionMember', ['user' => $user, 'passwd' => "Nik"], function ($m) use ($user) {
+            $m->to($user->email, $user->name)->subject('Welcome to ThyArt');
+        });*/
+
```
Comment (DeseineClement) [inline app/Http/Controllers/Api/UserController.php]: Ahah nope.
Please remove your comments

**Review by DeseineClement — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/Api/NewsletterController.php (hunk 1 above)
Comment (CorvinAlex) [reply app/Http/Controllers/Api/NewsletterController.php:67]: Yes

File: app/Http/Controllers/Api/NewsletterController.php (hunk 2 above)
Comment (CorvinAlex) [reply app/Http/Controllers/Api/NewsletterController.php]: done

File: app/Http/Controllers/Api/NewsletterController.php (hunk 3 above)
Comment (CorvinAlex) [reply app/Http/Controllers/Api/NewsletterController.php]: done

File: app/Http/Controllers/Api/UserController.php (hunk 4 above)
Comment (CorvinAlex) [reply app/Http/Controllers/Api/UserController.php]: done

File: app/Http/Controllers/Api/NewsletterController.php (hunk 1 above)
Comment (DeseineClement) [reply app/Http/Controllers/Api/NewsletterController.php:67]: ok

**Pull request merged**